README.md

```diff
@@ -83,7 +75,9 @@ Once a given scenario configured, it seemed useful to choose how the multi-partn
 
 Finally, with given scenarios and multi-partner learning approaches, we can address contributivity measurement approaches. See below sections [Configuration of contributivity measurement methods to be tested](#configuration-of-contributivity-measurement-methods-to-be-tested) and [Contributivity measurement approaches studied and implemented](#contributivity-measurement-approaches-studied-and-implemented).
 
-### Using the code files
+### Run an experiment
+
+#### Using the code files
```

Comment:
Maybe here, inbetween, a short text explaining that the intended usage is as a library etc., but it is also possible to run it the old fashion way etc.